docs/infrastructure/self-hosted/self-hosted-requirements.md

```diff
@@ -217,9 +219,52 @@ The following is an example architecture for the installation of the Mia-Platfor
 | Google Cloud Storage      |   SaaS      |                |      |      |          |
 | Google Cloud Platform KMS |   SaaS      |                |      |      |          |
 
-### Designating the Console Super User
 
-When installing an instance of Mia-Platform Console, a user with the Console Super User role will be added to the database automatically.
+## Runtime requirements 
+
+Applications developed with Mia-Platform Console will run in a Kubernetes environment. This Kubernetes runtime environment will be managed and monitored by Mia-Platform Console.  
+In order to do this, the following requirements must be fulfilled for a self-hosted installation.
+
+### Self-Hosted runtime installation prerequisites
+
+The following tools will be installed in your runtime Kubernetes clusters if desired:  
+
+* (Optional) Mia Monitoring stack
+* (Optional) Mia Logging stack
+* Traefik Ingress Controller
+* (Optional) Velero for Disaster Recovery
+
+:::note
+These are minimum requirements for your cluster. These requirements have a scale factor that must be taken in account when estimating your cluster size.
+:::
+
+| **Tool**                     | **Namespace CPU limits** | **Namespace RAM limits** | **Scales with**                                               |
+|------------------------------|--------------------------|--------------------------|---------------------------------------------------------------|
+| Monitoring stack             | 1 core                   | 3 GiB                    | Number of cluster nodes, number of metrics                    |
+| Logging stack                | 0.5 core                 | 1 GiB                    | Number of cluster nodes, number of applications logs, traffic |
+| Traefik Ingress Controller   | 0.3 core                 | 300 MiB                  | Traffic                                                       |
+| Velero for Disaster Recovery | 0.2 core                 | 400 MiB                  | -                                                             |
+
+In total, the minimum requirements in order to host all the above tools in the Kubernetes cluster are about 2 cores and 5 GiB of memory. Above values will scale with the number of nodes, traffic, applications.
+
+Velero, in addition, needs a S3-like bucket storage for keeping the Kubernetes cluster backup.
+
+Grafana Mimir will be installed in the same Kubernetes cluster hosting Mia-Platform Console. The minimum requirements needed for Grafana Mimir are 2 cores and 4 GiB of memory.
+
+For a simple Mia-Platform Console Project with 3 environments, a minimum of 6 cores and 12 GiB of memory are needed. Of course this is just an indicative estimation and the right sizing depends on your application.
```

Comment:
You mean 6 cores and 12 GiB of memory only for the applicative pods? So, 6 cores and 12GiB + the "operation" stack detailed above, right?

Reply:
Yes exactly.